applications/launchpad/backend/src/main.rs

```diff
@@ -149,7 +157,12 @@ fn main() {
 fn on_event(evt: GlobalWindowEvent) {
     if let WindowEvent::Destroyed = evt.event() {
         info!("Stopping and destroying all tari containers");
-        thread::spawn(|| block_on(shutdown_all_containers("default".to_string(), &DOCKER_INSTANCE.clone())));
+        thread::spawn(|| {
+            block_on(shutdown_all_containers(
+                DEFAULT_WORKSPACE_NAME.to_string(),
+                &DOCKER_INSTANCE.clone(),
+            ))
+        });
         sleep(Duration::from_secs(5));
```

Comment:
Remove this. 
What's the purpose of spawning a thread, only to call thread::sleep?

Just block_on and wait for the task to complete.

Reply:
This is fixed